example/src/App.tsx

```diff
@@ -136,6 +136,12 @@ class App extends React.Component<AppProps, AppState> {
     }).catch((error) => {
       this.logOutput(`logConversion: error ${JSON.stringify(error)}\n`);
     });
+    Radar.logTermination();
+    Radar.logBackgrounding();
```

Comment:
I think this should be in the following listener: https://capacitorjs.com/docs/apis/app#addlistenerpause-

but only on iOS (we automatically do this on Android)